src/main/java/eu/yals/Endpoint.java

```diff
@@ -13,45 +10,148 @@ private Endpoint() {
         throw new UnsupportedOperationException("Utility class");
     }
 
-    @RequestMethod(value = HttpMethod.POST, api = true)
-    public static final String STORE_API = "/api/store";
+    /**
+     * Endpoints accessible via browser
+     */
+    public static class UI {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'UI' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)